core/src/main/java/io/aiven/kafka/tieredstorage/metrics/Metrics.java

```diff
@@ -102,6 +106,26 @@ public Metrics(final Time time) {
         segmentFetchRequestedBytes.add(
             metrics.metricName("segment-fetch-requested-bytes-total", metricGroup),
             new CumulativeSum());
+
+        final var objectUploadRequestsRate =
+            new MetricNameTemplate("object-upload-rate", metricGroup, "", "object-type");
```

Comment:
Could we add a description here?

Reply:
Good catch. Others metrics have no description neither, we could tackle this on a separate PR I think.

Reply:
https://github.com/aiven/tiered-storage-for-apache-kafka/issues/313